### PR TITLE
Connection parameters

### PR DESCRIPTION
**Pull request by rudolfce**
Hello again!

I've done an experiment on trying to add parameters to the pymongo connection call. Unfortunatelly, I don't have a remote machine to test settings with, though I can say it attempts connection to the host either way. Port setting works as expected. Please, point out to any changes you think would be necessary.

### CODE REVIEW

**Comment by rudolfce** [comment]
Just noticed I messed up the commit history. I'll check this out later, but meawhile I think you could take a look either way

File: djongo/base.py [hunk 1]
```diff
@@ -85,20 +85,21 @@ def is_usable(self):
...
-            # from django.core.exceptions import ImproperlyConfigured
-            # raise ImproperlyConfigured(
-            #     "settings.DATABASES is improperly configured. "
-            #     "Please supply the NAME value.")
-        return self.settings_dict['NAME']
-
-    def get_new_connection(self, db_name):
+        settings_dict = {}
+
+        settings_dict['name'] = self.settings_dict.get('NAME') or 'djongo_test'
```
Comment (nesdis) [inline djongo/base.py]: Can u change this line and others to:

```
settings_dict['name'] = self.settings_dict.get('NAME', 'djongo_test')
```

File: djongo/base.py [hunk 2]
```diff
@@ -85,20 +85,21 @@ def is_usable(self):
...
+
+        return settings_dict
+
+    def get_new_connection(self, settings_dict):
         """
         This needs to be made more generic to accept
         other MongoClient parameters.
         """
-        return MongoClient()[db_name]
+        NAME = settings_dict.pop('name')
```
Comment (nesdis) [inline djongo/base.py]: Any reason to have NAME in all caps? lower letters should be ok?

Comment (rudolfce) [reply djongo/base.py]: I was attempting to please pylint. I'll change it for lower, no problem.

File: djongo/base.py (hunk 1 above)
Comment (rudolfce) [reply djongo/base.py]: Nice, I wasn't aware of that usage of the get method. I'll do it.

**Comment by rudolfce** [comment]
Very nice! I'll apply the requested changes as soon as I get access to my pc.

Also, I understand you rebased the repository, is this correct?

**Comment by nesdis** [comment]
Yes the repository has been rebased and multiple commits squashed...

**Comment by rudolfce** [comment]
Hello. Sorry for the delay. Just had a really crazy week.

I'll apply the changes soon. Thanks!

**Comment by rudolfce** [comment]
Hello again. I added a test case for the DatabaseWrapper class with two tests for the parameters and a mocked test for the invocation of the mongo engine. I noticed this is not the standard you currently use, so I would like to know what you think about it.

If you like it, I can help you add more tests for other parts of the module.

**Pull request merged**